spec/Cabin/Infotainment.vspec

```diff
@@ -189,6 +189,21 @@ HMI.DayNightMode:
   allowed: ['DAY', 'NIGHT']
   description: Current display theme
 
+HMI.IsScreenAlwaysOn:
+  datatype: boolean
+  type: actuator
+  description: Used to prevent the screen going black if no action placed.
+
+HMI.LastActionTime:
+  datatype: string
+  type: sensor
+  description: Time for last hmi action, formatted according to ISO 8601 with UTC time zone.
+
+HMI.DisplayOffTime:
+  datatype: string
+  type: actuator
+  description: Time used to make display off.
```

Comment:
In a previous comment (no longer visible becuase of later commits) Umang said:

_we can have time as absolute time, DisplayOffTime will have the time specification value. so HMI.LastActionTime+ Displayofftime = time (when display will go to off state)._ 

I have no strong opinion on what fits best for the use case, but I think the signal could benefit from some clarification. Either, if we want absolute time, like the display shall go off at 2023-04-18 14.00 UTC then we can possibly rephrase to something like:

```
HMI.DisplayOffTime:
  datatype: string
  type: actuator
  description: Time when display shall be turned off, formatted according to ISO 8601 with UTC time zone.
  comment: If DisplayOffTime is in the past and IsScreenAlwaysOn==False then display shall be off
```

But maybe it is more practical to have it as duration, possibly like:

```
HMI.DisplayOffDuration:
  datatype: uint16
  type: actuator
  unit: seconds
  description: Duration in seconds before the display is turned off. Value shall be 0 if screen never shall turn off.
  comment: Display shall be turned off at HMI.LastActionTime + HMI.DisplayOffDuration, unless HMI.IsScreenAlwaysOn==True
```

Let us bring it up to get feedback at the meeting today.